var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -11,17 +11,16 @@ class PyMaestrowf(PythonPackage):
        simulation studies."""
 
     homepage = "https://github.com/LLNL/maestrowf/"
-    url      = "https://github.com/LLNL/maestrowf/archive/v1.1.2.tar.gz"
+    git      = "https://github.com/LLNL/maestrowf.git"
```

Comment:
Ok, so after having thought about it, i think i am sure this is not a good idea. by changing the default download method to git, we're forcing all users to set up proper ssh keys. whereas, by leaving it as path to tar gets around the problem, and makes the tool completely public!

here and below, we should have sha hash of all the downloadable releases. use git only for the branches and bug-fixes, like @FrankD412  had before.